File: Python/tests/test_ra_noLA.py
import pytest

from hbayesdm.models import ra_noLA


def test_ra_noLA():
    _ = ra_noLA(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_ra_noLA.R
context("Test ra_noLA")
library(hBayesDM)

test_that("Test ra_noLA", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(ra_noLA(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
